client/state/sites/selectors.js

```diff
@@ -503,6 +503,10 @@ export function getSitePlan( state, siteId ) {
 	return site.plan;
 }
 
+export function getSitePlanSlug( state, siteId ) {
```

Comment:
yes, I thought the same but then I though that we should move all site selectors to into global ones since if not we will have site selectors in two different locations.
We could work on another PR just for this.